### DISCUSSION TITLE
Forge support

### DISCUSSION THREAD

**Discussion by Skyyofficial** [Okazakee/mcserver-lazymc-docker#20] [Ideas]
Is there a way to add forge support? I wanted to use this because many big and ressource hungry modpacks are on forge.
I tried to make it work but tbh I have no idea what I'm doing with docker ect.

**Comment by Okazakee** [comment]
I'm a bit constrained by free time, but going forward to it.

**Comment by Okazakee** [comment]
Just updated, please test and let me know!
